pkg/capabilities/v2/protoc/pkg/templates/server.go.tmpl

```diff
@@ -52,7 +52,7 @@ type {{.GoName}}Capability interface {
     Initialise(ctx context.Context, config string, telemetryService core.TelemetryService, store core.KeyValueStore, errorLog core.ErrorLog, pipelineRunner core.PipelineRunnerService, relayerSet core.RelayerSet, oracleFactory core.OracleFactory) error
 }
 
-func New{{.GoName}}Server(capability {{.GoName}}Capability) loop.StandardCapabilities {
+func New{{.GoName}}Server(capability {{.GoName}}Capability) *{{.GoName|LowerFirst}}Server {
```

Comment:
Nit: rename the struct to be upper case, it's not great to return non-public types.